### PR TITLE
changes for cosmos version upgarde

### PR DESCRIPTION
**Pull request by prashantkrv2020**

### CODE REVIEW

**Comment by prashantkrv2020** [comment]
ProtocolVersionSetter to initialize upgradeKeeper not implemented yet

**Review by dheerajkd30 — DISMISSED** [review]

**Comment by dheerajkd30** [comment]
> ProtocolVersionSetter to initialize upgradeKeeper not implemented yet

Have raised a PR for this issue  and other build issues in PR https://github.com/comdex-official/comdex/pull/33.

**Review by dheerajkd30 — DISMISSED** [review]

**Review by dheerajkd30 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -12,15 +12,36 @@ LD_FLAGS := -s -w \
...
+.PHONY: all install build verify
+
 .PHONY: clean
 clean:
 	rm -rf ./bin ./vendor
 
-.PHONY: install
+all: verify build
+
+install:
```
Comment (bsrinivas8687) [inline Makefile:31]: Why two install commands? Can you please remove one

File: Makefile [hunk 2]
```diff
@@ -12,15 +12,36 @@ LD_FLAGS := -s -w \
...
+ifeq (${OS},Windows_NT)
+	go build -mod=readonly ${BUILD_FLAGS} -o ${GOBIN}/comdex.exe ./node
+else
+	go build -mod=readonly ${BUILD_FLAGS} -o ${GOBIN}/comdex ./node
+endif
+
+build:
```
Comment (bsrinivas8687) [inline Makefile:38]: This is same as install command so not needed

**Review by bsrinivas8687 — APPROVED** [review]

**Pull request merged**